prow/pubsub/subscriber/subscriber.go

```diff
@@ -154,16 +165,65 @@ func (peh *periodicJobHandler) getProwJobSpec(cfg *config.Config, pe ProwJobEven
 }
 
 // presubmitJobHandler implements jobHandler
-type presubmitJobHandler struct{}
+type presubmitJobHandler struct {
+	GitClient git.ClientFactory
```

Comment:
This git client is never instantiated in the `handleMessage` function.

Reply:
good catch, adde